src/app.js

```diff
@@ -1,14 +1,17 @@
 require('dotenv').config();
 const bodyParser = require('body-parser');
 const express = require('express');
-const octokit = require('@octokit/rest');
+const Octokit = require('@octokit/rest');
 const nacl = require('tweetnacl');
 nacl.util = require('tweetnacl-util');
 
 const username = 'yourusername';  // TODO: your GitHub username here
```

Comment:
Don't forget you'll have to change this to your actual username or an environment variable containing it to actually get it to work.